src/ddmd/s2ir.d

```diff
@@ -491,7 +492,7 @@ extern (C++) class S2irVisitor : Visitor
     override void visit(SwitchStatement s)
 //    { .visit(irs, s); }
     {
-        int string;
+        bool isString;
```

Comment:
I was confused by this at first, because it looks like a member function name. I prefer to use short, all lower case local names to distinguish them.